### ISSUE TITLE
jcarouselAutoscroll does not work when you have 2 carousel simultaneously

### ISSUE THREAD

**Issue by kafeltz** [jsor/jcarousel#686]
If you have 2 separated div with their own class (divs). The plugin will always return the first jcarousel instance.

I could not make it work on fiddle, but you can see in this temporary page: http://haai.com.br/paroquia/

On jcarousel plugin source code i put a console.log(this) to show the instance is wrong.

I have two separated carousel with:
-        var jcarousel = $('.jcarousel');
-         var jcarousel = $('.jcarousel-banner'); // here the instance 'this' comes the wrong one

tldr: it auto-start-scrolling  the wrong caroussel!

this: $('.jcarousel-banner');
auto-starts the other one: $('.jcarousel');
